boards/arm/stm32/stm32f4discovery/configs/adb/defconfig

```diff
@@ -44,21 +44,25 @@ CONFIG_INTELHEX_BINARY=y
 CONFIG_LIBC_ENVPATH=y
 CONFIG_LIBC_EXECFUNCS=y
 CONFIG_LIBUV=y
-CONFIG_LIBUV_PIPE=y
-CONFIG_LIBUV_STREAM=y
 CONFIG_MMCSD=y
+CONFIG_NET=y
+CONFIG_NETDEV_LATEINIT=y
+CONFIG_NET_SOCKOPTS=y
+CONFIG_NET_TCP_NO_STACK=y
+CONFIG_NET_UDP_NO_STACK=y
```

Comment:
remove NO_STACK?

Reply:
```bash
netlib_setifstatus.c: In function 'netlib_ifup':
netlib_setifstatus.c:66:44: error: 'NET_SOCK_TYPE' undeclared (first use in this function)
   66 |       int sockfd = socket(NET_SOCK_FAMILY, NET_SOCK_TYPE, NET_SOCK_PROTOCOL);
      |                                            ^~~~~~~~~~~~~
netlib_setifstatus.c:66:44: note: each undeclared identifier is reported only once for each function it appears in
netlib_getifstatus.c: In function 'netlib_getifstatus':
netlib_getifstatus.c:68:44: error: 'NET_SOCK_TYPE' undeclared (first use in this function)
   68 |       int sockfd = socket(NET_SOCK_FAMILY, NET_SOCK_TYPE, NET_SOCK_PROTOCOL);
      |                                            ^~~~~~~~~~~~~
netlib_getifstatus.c:68:44: note: each undeclared identifier is reported only once for each function it appears in
netlib_setifstatus.c: In function 'netlib_ifdown':
netlib_setifstatus.c:109:44: error: 'NET_SOCK_TYPE' undeclared (first use in this function)
  109 |       int sockfd = socket(NET_SOCK_FAMILY, NET_SOCK_TYPE, NET_SOCK_PROTOCOL);
      |                                            ^~~~~~~~~~~~~
make[3]: *** [/home/huang/Work/nx/apps/Application.mk:153: netlib_setifstatus.c.home.huang.Work.nx.apps.netutils.netlib.o] Error 1
make[3]: *** Waiting for unfinished jobs....
make[3]: *** [/home/huang/Work/nx/apps/Application.mk:153: netlib_getifstatus.c.home.huang.Work.nx.apps.netutils.netlib.o] Error 1
CC:  libuv/src/uv-data-getter-setters.c
CC:  libuv/src/version.c
make[3]: Leaving directory '/home/huang/Work/nx/apps/system/readline'
CC:  libuv/src/idna.c
netlib_setipv4addr.c: In function 'netlib_set_ipv4addr':
netlib_setipv4addr.c:66:44: error: 'NET_SOCK_TYPE' undeclared (first use in this function)
   66 |       int sockfd = socket(NET_SOCK_FAMILY, NET_SOCK_TYPE, NET_SOCK_PROTOCOL);
      |                                            ^~~~~~~~~~~~~
netlib_setipv4addr.c:66:44: note: each undeclared identifier is reported only once for each function it appears in
make[3]: *** [/home/huang/Work/nx/apps/Application.mk:153: netlib_setipv4addr.c.home.huang.Work.nx.apps.netutils.netlib.o] Error 1
make[3]: Leaving directory '/home/huang/Work/nx/apps/netutils/netlib'
make[2]: *** [Makefile:42: /home/huang/Work/nx/apps/netutils/netlib_all] Error 2
make[2]: *** Waiting for unfinished jobs....
```

Reply:
since the new libuv porting doesn't depend on net now, let's remove net config instead.